manager/integration/tests/test_recurring_job.py

```diff
@@ -759,3 +794,641 @@ def backup_complete_predicate(b):
     # the backups
     vol.recurringUpdate(jobs=[])
     backupstore.backupstore_wait_for_lock_expiration()
+
+
+@pytest.mark.recurring_job  # NOQA
+def test_recurring_job_groups(set_random_backupstore, client, batch_v1_beta_api):  # NOQA
+    """
+    Scenario: test recurring job groups (S3/NFS)
+    """
+    volume1_name = "test-job-1"
+    volume2_name = "test-job-2"
+    client.create_volume(name=volume1_name, size=SIZE)
+    client.create_volume(name=volume2_name, size=SIZE)
+    volume1 = common.wait_for_volume_detached(client, volume1_name)
+    volume2 = common.wait_for_volume_detached(client, volume2_name)
+
+    self_host = get_self_host_id()
+
+    volume1.attach(hostId=self_host)
+    volume2.attach(hostId=self_host)
+    volume1 = wait_for_volume_healthy(client, volume1_name)
+    volume2 = wait_for_volume_healthy(client, volume2_name)
+
+    snap1 = SNAPSHOT + "1"
+
+    back1 = BACKUP + "1"
+
+    group1 = "group-1"
+    group2 = "group-2"
+
+    recurring_jobs = {
+        snap1: {
+            TASK: SNAPSHOT,
+            GROUPS: [group1, group2],
+            CRON: "* * * * *",
+            RETAIN: 2,
+            CONCURRENCY: 2
+        },
+        back1: {
+            TASK: BACKUP,
+            GROUPS: [group1],
+            CRON: "* * * * *",
+            RETAIN: 1,
+            CONCURRENCY: 2
+        },
+    }
+    create_recurring_jobs(client, recurring_jobs)
+
+    volume1.recurringJobAdd(name=group1, isGroup=True)
```

Comment:
Python method name should be snake case

Reply:
This POST queried for [actions](https://github.com/longhorn/longhorn-manager/blob/master/api/router.go#L68-L95).

Reply:
Thanks :), although this does not follow up with the language convention, that's fine for now based on our own convention.